README.rst

```diff
@@ -82,6 +82,13 @@ For example:
     [flake8]
     banned-modules = decimal.Decimal = Use ints and floats only.
 
+Entries ending in ``.*`` are treated as wildcards; the example will ban both ``import foo`` and ``import foo.bar``:
```

Comment:
A little wordsmithing - unnecessary semicolon
```suggestion
Entries ending in ``.*`` are treated as wildcards. For example, to ban both ``import foo`` and ``import foo.bar``:
```

Reply:
Out of date, resolving